README.md

```diff
@@ -295,6 +295,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [LoginLlama](https://loginllama.app) - A login security API to detect fraudulent and suspicious logins and notify your customers. Free for 1,000 logins per month.
   * [MailboxValidator](https://www.mailboxvalidator.com) — Email verification service using real mail server connection to confirm valid email. Free API plan has 300 verifications per month.
   * [Meteosource Weather API](https://www.meteosource.com/) — global weather API for current and forecasted weather data. Forecasts are based on a machine learning combination of more weather models to achieve better accuracy. Free plan comes with 400 calls per day.
+  * [microcks.io](https://microcks.io/) – It is a platform for turning your API and microservices assets - OpenAPI specs, AsyncAPI specs, gRPC protobuf, GraphQL schema, Postman collections, SoapUI projects - into live mocks in seconds.
```

Comment:
The tick boxes in the PR block needs to be ticked - meaning your entry should meet those criteria.

I dont think this one fits the bill.

Reply:
Agree, I will drop the PR.

Reply:
thank you!